app/src/main/java/com/remindme/app/ui/component/CustomViews.kt

```diff
@@ -52,133 +49,172 @@ import com.remindme.app.ui.theme.Grey700
+                null
+            )
+        }
     }
-  }
 
-  OutlinedTextField(
-    value = text,
-    onValueChange = { text = it },
-    singleLine = true,
-    maxLines = 1,
-    readOnly = true,
-    label = {
-      Row {
-        Text(label, color = Grey700)
-        Box(
-          modifier = Modifier
-            .fillMaxWidth(),
-          contentAlignment = Alignment.BottomEnd
-        ) { Text("03-07-2022", color = Grey700) }
-      }
-    },
-    colors = TextFieldDefaults.outlinedTextFieldColors(
-      focusedBorderColor = MaterialTheme.colors.primary,
-      unfocusedBorderColor = MaterialTheme.colors.primary,
-      focusedLabelColor = MaterialTheme.colors.primary,
-      cursorColor = MaterialTheme.colors.primary,
-      textColor = MaterialTheme.colors.primary
-    ),
-    modifier = Modifier.fillMaxWidth(),
-    trailingIcon = trailingIconView
-  )
+    OutlinedTextField(
+        value = text,
+        onValueChange = { text = it },
+        singleLine = true,
+        maxLines = 1,
+        readOnly = true,
+        label = {
+            Row {
+                Text(label, color = Grey700)
+                Box(
+                    modifier = Modifier
+                        .fillMaxWidth(),
+                    contentAlignment = Alignment.BottomEnd
+                ) { Text("03-07-2022", color = Grey700) }
+            }
+        },
+        colors = TextFieldDefaults.outlinedTextFieldColors(
+            focusedBorderColor = MaterialTheme.colors.primary,
+            unfocusedBorderColor = MaterialTheme.colors.primary,
+            focusedLabelColor = MaterialTheme.colors.primary,
+            cursorColor = MaterialTheme.colors.primary,
+            textColor = MaterialTheme.colors.primary
+        ),
+        modifier = Modifier.fillMaxWidth(),
+        trailingIcon = trailingIconView
+    )
 }
 
 @Composable
-fun RemindBeforeCheckboxGroup() {
-  Column(modifier = Modifier.padding(top = 20.dp)) {
-    Row {
-      CheckboxWithLabel(label = "60 days")
-      Spacer(Modifier.size(width = 50.dp, height = 0.dp))
-      CheckboxWithLabel(label = "7 days")
+fun RemindBeforeCheckboxGroup(modifier: Modifier = Modifier) {
+    Column(modifier = modifier.padding(top = 10.dp)) {
+        Row {
+            CheckboxWithLabel(label = "60 days")
+            Spacer(Modifier.size(width = 50.dp, height = 0.dp))
+            CheckboxWithLabel(label = "7 days")
+        }
+        Row {
+            CheckboxWithLabel(label = "30 days")
+            Spacer(Modifier.size(width = 50.dp, height = 0.dp))
+            CheckboxWithLabel(label = "1 day")
+        }
     }
-    Row {
-      CheckboxWithLabel(label = "30 days")
-      Spacer(Modifier.size(width = 50.dp, height = 0.dp))
-      CheckboxWithLabel(label = "1 day")
+}
+
+@Composable
+fun NotifyRadioGroupOptions(
+    modifier: Modifier = Modifier,
+    radioOptions: List<String> = listOf("Weekly", "Daily", "Default")
```

Comment:
we can create string array in resource instead of hardcoding here.

Reply:
done